settings.gradle.kts

```
@@ -29,7 +29,7 @@ dependencyResolutionManagement {
         google()
         mavenCentral()
         // Uncomment the following if testing snapshots from Maven Central
-        // maven("https://s01.oss.sonatype.org/content/repositories/snapshots")
+        maven("https://s01.oss.sonatype.org/content/repositories/snapshots")
```

needs to be recommented again
```suggestion
        // maven("https://s01.oss.sonatype.org/content/repositories/snapshots")
```